### PR TITLE
A blogpost for Season of Docs

### PR DESCRIPTION
**Pull request by aijamalnk**
Blog post contains the ideas list that Apache Beam is submitting for Season of Docs.

### CODE REVIEW

**Comment by aijamalnk** [comment]
r: @kennknowles If you don't mind looking into it. Thank you!

**Comment by pabloem** [comment]
this LGTM.

**Comment by kennknowles** [comment]
http://apache-beam-website-pull-requests.storage.googleapis.com/8365/index.html

File: website/src/_posts/2019-04-19-season-of-docs.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Apache Beam does have help from excellent technical writers, but the documentation needs of the project often exceed their bandwidth. This is why we are excited about this program.
+
+After discussing ideas in the community, we have been able to find mentors, and frame two ideas that we think would be a great fit for an incoming tech writer to tackle. We hope you will find this opportunity interesting - and if you do, please get in touch by emailing the Apache Beam mailing list at [dev@beam.apache.org](mailto:dev@beam.apache.org) (you will need to subscribe first by emailing to [dev-subscribe@beam.apache.org](mailto:dev-subscribe@beam.apache.org)).  
+
+The project ideas available in Apache Beam are described below. Please take a look and ask any questions that you may have. We will be very happy to help you get onboarded with the project.
+
+### Project ideas
+
+**Deployment of a Flink and Spark Clusters with Portable Beam**
```
Comment (kennknowles) [inline website/src/_posts/2019-04-19-season-of-docs.md]: Grammar & rephrase idea: "Deployment of Flink and Spark Clusters for use with Portable Beam"

File: website/src/_posts/2019-04-19-season-of-docs.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+The Apache Beam vision has been to provide a framework for users to write and execute pipelines on the programming language of your choice, and the runner of your choice. As the reality of Beam has evolved towards this vision, the way in which Beam is run on top of runners such as Apache Spark and Apache Flink has changed.
+
+These changes are documented in the wiki and in design documents, and are accessible for Beam contributors; but they are not available in the user-facing documentation. This has been a barrier of adoption for other users of Beam.
+
+This project involves improving the Flink Runner page[1] to include strategies to deploy Beam on a few different environments: A Kubernetes cluster, a Google Cloud Dataproc cluster, and an AWS EMR cluster. There are other places in the documentation that should be updated in this regard[4][5].
```
Comment (kennknowles) [inline website/src/_posts/2019-04-19-season-of-docs.md]: On the web, it looks more natural to link the text itself. I think in Markdown you can do this via:

```
... improving the [Flink Runner page] ...

[Flink Runner page](...)
```

But if that doesn't work with Jekyll, some other ideas. You can link these directly inline. But I do think the bullet list of links at the end is nice to read. So you could just have inline links and then a transition sentence, like "Here are the pages proposed for revision:"

**Comment by kennknowles** [comment]
Checked the logs for the Website PreCommit. It was an external site not responding. No internal errors.

**Pull request merged**